cmd/kubeadm/app/componentconfigs/kubelet_test.go

```diff
@@ -225,6 +231,40 @@ func TestKubeletDefault(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:       "ContainerRuntimeEndpoint defaulting works",
+			clusterCfg: kubeadmapi.ClusterConfiguration{},
+			nodeCfg: kubeadmapi.NodeRegistrationOptions{
+				CRISocket: "unix:///var/run/containerd/containerd.sock",
```

Comment:
Should the "canonical" form be used `/run`?

Reply:
This code is only for unit test, so I don't think it needs to be modified.

Reply:
Thanks, was just curious because I noticed the kubeadm join always adding /var/run to the kubeadm-flags.envs on the worker-node instead what was in the kubeadm init config /run/.